tests/system/test_system.py

```diff
@@ -358,11 +358,14 @@ def tearDownClass(cls):
         cls._db.drop()
 
     def setUp(self):
+        self.instances_to_delete = []
         self.to_delete = []
 
     def tearDown(self):
         for doomed in self.to_delete:
             doomed.drop()
+        for instance in self.instances_to_delete:
+            instance.delete()
```

Comment:
By having this logic here, we have to create and delete an instance for every test that needs a multi-region instance. This is likely going to increase the number of `RESOURCE_EXHAUSTED` errors we see.

I think moving the instance creation into `setUpClass` similar to how `cls._db` is done would be better. That way the tests can all reuse the instance :+1: